src/electron/electron/main/services/WindowService.ts

```diff
@@ -307,39 +345,65 @@ export class WindowService {
     const trayImage = nativeImage.createFromPath(appIcon);
     trayImage.setTemplateImage(true);
     this.tray = new Tray(trayImage);
-    this.updateTray();
+    await this.updateTray();
   }
 
-  private getTrayMenuTemplate(): MenuItemConstructorOptions[] {
-    const versionAndUpdate: MenuItemConstructorOptions[] = [
+  private async getTrayMenuTemplate(): Promise<MenuItemConstructorOptions[]> {
+    const settings: Settings = await Settings.findOne({ where: { onlyOneEntityShouldExist: 1 } });
+    
+    const versionUpdateAndStudyInfo: MenuItemConstructorOptions[] = [
       { label: `Version ${app.getVersion()}`, enabled: false },
       {
         label: 'Check for updates',
         enabled: false,
         click: () => this.appUpdaterService.checkForUpdates({ silent: false })
       },
-      { type: 'separator' }
+      { type: 'separator' },
+      { 
+        label: 'Copy Subject Id',
```

Comment:
super minor, but i suggest to move the "copy"-feature below the "listing the subject ID" item